### DISCUSSION TITLE
two emqx broker connected via bridging.. are messages queued and replied?

### DISCUSSION THREAD

**Discussion by slaesh** [emqx/emqx#9169] [Q&A] [answered]
if these two brokers are connected but due to some ethernet problems they loose the connection.. for a few minutes, maybe even multiple hours?

what happens if the re-connect..

do they replay all occurred messages?

**Comment by lafirest** [comment, marked as answer]
Yes, in the bridge action of the rule engine you can turn on disk caching and it will queue messages. But I think you'd better test it out yourself

**Reply by slaesh** [reply]
Will try that, thanks!
